.github/workflows/rust.yml

```diff
@@ -15,7 +15,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Rust fmt
-      run: cargo fmt -- --check
+      run: |
+        cargo fmt -- --check
+        rustfmt **/*.rs --check
```

Comment:
Maybe you need to split this into two separate `run` invocations?

Reply:
That's not what causes the error.